Makefile

```diff
@@ -1,14 +1,11 @@
-PACKAGE = github.com/named-data/YaNFD
 VERSION = 1.3.0.0
-# COMMIT = git rev-parse --short HEAD
-# DATE != date
 
 .PHONY: all install clean test coverage
 
 all: yanfd
 
 yanfd: clean
-	go build -ldflags "-X 'main.Version=${VERSION}'" ${PACKAGE}/cmd/yanfd
```

Comment:
If you remove `"-X 'main.Version=${VERSION}'"` the Version will be empty

Reply:
Didn't know this, thanks! Reverted.